server/src/routes/pollRoute.ts

```diff
@@ -72,7 +75,10 @@ pollRouter.get("/status", async (req, res) => {
 pollRouter.get("/result", async (req, res) => {
   const { pollId } = req.query;
   try {
-    const result = await getResult(pollId);
+    if (typeof req.headers.utorid !== "string") {
```

Comment:
Why are you checking for `utorid` here? it should be be ` req.query.pollId`

Reply:
Sorry my bad